internal/dump/zip.go

```diff
@@ -39,6 +39,7 @@ func (d *Dumper) dumpNodeZip(ctx context.Context, node *restic.Node, zw *zip.Wri
 		Modified:           node.ModTime,
 	}
 	header.SetMode(node.Mode)
+	header.Method = zip.Deflate
```

Comment:
I would set this only for regular files, or at least not for symlinks.

Reply:
sounds reasonable, symlinks are probably usually too short to benefit from compression.